cparser/Elab.ml

```diff
@@ -1237,16 +1259,14 @@ and elab_item zi item il =
       begin match elab_string_literal loc w s, unroll env ty_elt with
       | CStr s, TInt((IChar | ISChar | IUChar), _) ->
           if not (I.index_below (Int64.of_int(String.length s - 1)) sz) then
-            warning loc "initializer string for array of chars %s is too long"
-                        (I.name zi);
+            warning loc Unnamed "initializer-string for array of chars is too long";
```

Comment:
"initializer-string" sure doesn't sound like proper English to me.  Amazingly that's what GCC and Clang use.  What next? "run-together-words"?